src/backend/columnar/columnar_customscan.c

```diff
@@ -267,11 +267,6 @@ ColumnarSetRelPathlistHook(PlannerInfo *root, RelOptInfo *rel, Index rti,
 		return;
 	}
 
-	/*
-	 * Here we want to inspect if this relation pathlist hook is accessing a columnar table.
-	 * If that is the case we want to insert an extra path that pushes down the projection
-	 * into the scan of the table to minimize the data read.
-	 */
```

Comment:
How is the removal of this comment related to the change we are making?

Reply:
right, actually it has nothing to do with the patch itself, let me revert this part :)